common-docs/identity/sign-in.md

```diff
@@ -4,12 +4,24 @@
 
 ## What is a Microsoft Account?
 
-A Microsoft Account, or MSA, is the personal user account you create when you want to sign into Microsoft services like outlook.com, Xbox, Skype, etc. Read more [here](https://aka.ms/AAdd6f8).
+A Microsoft Account, or MSA, is the personal user account you create when you want to sign into Microsoft services like outlook.com, Xbox, Skype, etc. Read more [here](https://aka.ms/AAdd6f8). We also support Microsoft 365 accounts that you can use from school or your workplace.
 
 ## How does it work in MakeCode?
 
 A number of new features are coming that will only be available to signed-in users. These features need to know your identity so they can do things for you like store data in the cloud, and connect you to other users. The first of these features is one we're calling [Cloud Sync](/identity/cloud-sync). With Cloud Sync, when you're signed into MakeCode, we'll store your projects in the cloud where they're accessible from anywhere.
 
+Click the sign in button in the upper right corner to log into MakeCode.
+
+![Home screen sign in button](/static/identity/sign-in-button.jpg)
+
+Watch this video for a demonstration of creating a Microsoft Account and signing in to MakeCode:
+
+https://youtu.be/4r5tGfi2BFI
+
+Once you've signed in, you'll see your login initials displayed in the upper-right corner of the editor.
```

Comment:
Small thing: We show login initials only if the account doesn't have a profile pic.